storage/replica_test.go

```diff
@@ -1550,13 +1550,27 @@ func TestRangeSequenceCacheStoredTxnRetryError(t *testing.T) {
 		return err
 	}
 
-	if firstErr := try(); firstErr != nil {
-		t.Fatal(firstErr)
+	if err := try(); err != nil {
+		t.Fatal(err)
 	}
 	txn.Timestamp.Forward(txn.Timestamp.Add(10, 10)) // can't hurt
-	secondErr := try()
-	if _, ok := secondErr.(*roachpb.TransactionRetryError); !ok {
-		t.Fatal(secondErr)
+	err := try()
+	if _, ok := err.(*roachpb.TransactionRetryError); !ok {
+		t.Fatal(err)
+	}
+
+	//  Pretend we restarted by increasing the epoch. We didn't increase
+	//  the sequence though, so still the same error.
```

Comment:
Why does that still get the same error? Aren't sequence numbers only valid during a single epoch?